bahmni_api_feed/__manifest__.py

```diff
@@ -20,8 +20,7 @@
         'views/order_type_shop_map_view.xml',
         'views/res_users_view.xml',
         'data/mrs_person_attributes_data.xml',
-        'views/menus.xml',
-        'data/syncable_units_mapping.xml'
```

Comment:
Have you verified whether removing this will cause any errors on an existing deployment?

Reply:
I verified the same @mohan-13, the xml file is to map the units and the same is handled on the initializer side. The changes works fine.